addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -128,19 +128,25 @@ export class LinkPlugin extends Plugin {
                 this.handleAutomaticLinkInsertion();
             }
         });
-        this.services.command.add(
+        this.removeLinkShortcut = this.services.command.add(
             "Create link",
             () => {
                 this.toggleLinkTools();
             },
             {
                 hotkey: "control+k",
+                // TODO: the category file should be moved from web_editor to html_editor
```

Comment:
Move it in this commit and add a test to be sure. "Create link" is the first command in the command palette